lib/locales/es.ts

```diff
@@ -231,19 +206,20 @@ export const locale: Locale = {
     blaise_of_sebaste_bishop: 'San Blás, Obispo y Mártir',
     bonaventure_of_bagnoregio_bishop: 'San Buenaventura, Obispo y Doctor',
     boniface_of_mainz_bishop: 'San Bonifacio, Obispo y Mártir',
+    brigid_of_kildare_virgin: 'Santa Brígida de Kildare, Abadesa',
     brigid_of_kildare_virgin_copatroness_of_ireland: 'Santa Brígida de Kildare, Vírgen y Patrona Secundaria de Irlanda',
     bruno_of_cologne_priest: 'San Bruno, Sacerdote',
     carlos_manuel_rodriguez_santiago: 'Beato Carlos Manuel Rodriguez',
     casimir_of_poland: 'San Casimiro',
     catherine_of_alexandria_virgin: 'Santa Catalina de Alexandria, Vírgen y Mártir',
     catherine_of_saint_augustine_de_simon_de_longpre_virgin: 'Beata María Catalina de San Agustín, Vírgen',
     catherine_of_siena_virgin: 'Santa Catalina de Siena, Vírgen y Doctor de la Iglesia',
-    catherine_of_siena_virgin_copatroness_of_europe:
-      'Santa Catalina de Siena, Vírgen, Doctor de la Iglesia y Patrona Secundaria de Europa',
-    catherine_of_siena_virgin_copatroness_of_italy_and_europe:
-      'Santa Catalina de Siena, Vírgen, Doctor de la Iglesia y Patrona Secundaria de Italia y Europa',
+    catherine_of_siena_virgin_copatroness_of_europe: 'Santa Catalina de Siena, Vírgen, Doctor de la Iglesia y Patrona Secundaria de Europa',
```

Comment:
Ideally use `Doctora` instead of `Doctor` since it's a woman we are referring to.

Reply:
Thanks! I also fixed this in the following keys:
- `catherine_of_siena_virgin`;
- `catherine_of_siena_virgin_copatroness_of_europe`;
- `catherine_of_siena_virgin_copatroness_of_italy_and_europe`;
- `teresa_of_jesus_of_avila_virgin`;
- `therese_of_the_child_jesus_and_the_holy_face_of_lisieux_virgin`;
- `therese_of_the_child_jesus_and_the_holy_face_of_lisieux_virgin_copatroness_of_france`.